README.md

```diff
@@ -115,6 +115,14 @@ You can browse their respective docs on the Terraform Registry,
 [here](https://registry.terraform.io/providers/hashicorp/random/latest/docs)
 and [here](https://registry.terraform.io/providers/hashicorp/tls/latest/docs).
 
+### Usage of Terraform binary
```

Comment:
It might be good to shout that (I believe) this should only happen if the newer `--providers-schema` flag is not provided 👍

Reply:
Good shout. Added in a7667f2 :zap: